lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/data/179d_ashrae_90_1_2007.spc_typ.json

```diff
@@ -45,6 +45,7 @@
       "occupancy_schedule": "Nonres_Occ_Sch",
       "occupancy_activity_schedule": "ACM_FullServiceRestaurant_ACTIVITY_SCH",
       "is_residential": null,
+      "occupancy_fraction_sensible": 0.451,
```

Comment:
new field added on the google spreadsheet.

ACM gives Sensible Heat Gain and Latent Heat Gain (Btu/h).
Sensible + Latent = Total, convert that to Watts => this is the value of the "occupancy_activity_schedule"

Sensible / Total = occupancy_fraction_sensible